### ISSUE TITLE
Multi GPU usage during multi-label classification with XLNet

### ISSUE THREAD

**Issue by dbswndigh** [kaushaltrivedi/fast-bert#145]
I am following your [notebook](https://github.com/kaushaltrivedi/fast-bert/blob/master/sample_notebooks/new-toxic-multilabel.ipynb)

When I used single GPU with logging_steps=50, epochs=6, I could get reasonable results:
<img width="405" alt="스크린샷 2019-12-11 오후 2 15 50" src="https://user-images.githubusercontent.com/44551410/70594462-fa7ab400-1c23-11ea-8bbc-dacef3b2cbba.png">

**However, with multiGPU, the train loss after each epoch didn't decrease fast unlike with singleGPU. Could you explain me why this happens?**

**Comment by dbswndigh** [comment]
When I increase epochs with multiGPU, I could find that the loss decreases well too. 
Why loss decreases slowly with multiGPU seems related with learning rate/ optimizer issue.
